source/component/PasDoc_GenHtml.pas

```diff
@@ -2521,65 +2520,78 @@ function THTMLDocGenerator.MakeBodyBegin: string;
   begin
     Result := '';
 
-    if Title <> '' then
+    if (Title <> '')  then
     begin
       Result := Result + '<h2>' + LocalMakeLink('index.html', Title) + '</h2>';
-    end;
-
-    if Introduction <> nil then
+    end
+    else
     begin
-      if Introduction.ShortTitle = '' then
-        Result := Result + LocalMakeParagraphLink(Introduction.OutputFileName, trIntroduction) else
-        Result := Result + LocalMakeParagraphLink(Introduction.OutputFileName, Introduction.ShortTitle);
+      if (Introduction <> nil) then
+      begin
+        if Introduction.ShortTitle='' then
+          Result := Result + '<h2>' + LocalMakeLink('index.html', Introduction.Title) + '</h2>'
+        else
+          Result := Result + '<h2>' + LocalMakeLink('index.html', Introduction.ShortTitle) + '</h2>';
+      end;
```

Comment:
You changed too much here, I'm afraid. The previous review comments requested to link to `index.html` if you just want to link to the "home page". But an explicit link to "introduction" should still look like in the original, i.e. point to `Introduction.OutputFileName`.

Also, keep the formatting of the original code and don't make changes when you didn't want. Looking at this change:

- you changed `if Introduction <> nil then` to `if (Introduction <> nil) then`, we don't use such excessive parenthesis
- you removed spaces from `Introduction.ShortTitle = ''` comparison
- you left a comment `//if (Title = '') and` below

Please avoid changing code formatting -- we try to follow consistent Pascal coding standards in PasDoc. We're definitely not ideal (a lot of people worked on this project and there are pieces of code with inconsistent lines), but we try :)

Please clean this change a little -- you want to just change `LocalMakeParagraphLink` to `LocalMakeLink` as far as I see. As as I see no other change in code in case `Introduction <> nil` should be needed.

Reply:
Fixed